DragonLoop/DragonLoopAPI/Controllers/BusController.cs

```diff
@@ -85,6 +85,46 @@ public async Task<IActionResult> PutBus(int id, Bus bus)
             return NoContent();
         }
 
+		//PUT: api/Bus/5/UpdateBusLocation
+		///<summary>
+		/// This method will update a bus's location given an ID. The request will be made as follows:
+		/// content-type: application/json
+		/// {
+		///		XCoordinate : "1.00",
+		///		YCoordinate : "1.00"
+		/// }
+		/// </summary>
+		[HttpPut("{id}/UpdateBusLocation")]
+		public async Task<IActionResult> PutBusLocation(int id, Bus newBusCoord)
+		{
+			var existingBus = _context.Buses.Where(b => b.BusId == id).FirstOrDefault<Bus>();
```

Comment:
You can do something like this here
`var bus = await _context.Buses.FindAsync(id);`